model/registry/file.yaml

```diff
@@ -4,6 +4,41 @@ groups:
     display_name: File Attributes
     brief: "Describes file attributes."
     attributes:
+      - id: file.accessed
+        type: string
+        brief: >
+          Time when the file was last accessed, in ISO 8601 format.
+        note: >
+          This attribute might not be supported by some file systems — NFS, FAT32, in embedded OS, etc.
+        stability: experimental
+        examples: ['2021-01-01T12:00:00Z']
+      - id: file.attributes
+        type: string[]
+        brief: >
+          Array of file attributes.
+        note: >
+          Attributes names depend on the OS or file system. Here’s a non-exhaustive list of values expected for this
```

Comment:
Is it possible with semantic-conventions to have an array of enum values? If it is, can it be used here? 

There should be a relatively small set of possible attribute values among all well-known filesystems. If they're defined in an enum, it would avoid different users having different terms/spellings for the same attribute

Reply:
as of now we don't have a possibility to have an array of enum values. I have created an [enhancement](https://github.com/open-telemetry/weaver/issues/329) for it

Reply:
👍 thanks for creating the enhancement request